vb_suite/test_perf.py

```diff
@@ -56,7 +56,7 @@
 from pandas import DataFrame, Series
 
 from suite import REPO_PATH
-VB_DIR = os.path.dirname(os.path.abspath(__file__))
+VB_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
without this, I got following error:
```
$ ./test_perf.sh -b master -t HEAD -r series
Verifying specified commits exist in repo...
Traceback (most recent call last):
  File "/Users/junya/work/github/pandas/vb_suite/test_perf.py", line 606, in <module>
    r=git.Repo(VB_DIR)
  File "/usr/local/opt/pyenv/versions/pandas27/lib/python2.7/site-packages/git/repo/base.py", line 149, in __init__
    raise InvalidGitRepositoryError(epath)
git.exc.InvalidGitRepositoryError: /Users/junya/work/github/pandas/vb_suite
```

``VB_DIR`` is pointing ``vb_suite`` directory now.